app/views/components/feedback_frontend.scala.html

```diff
@@ -0,0 +1,34 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.components.gds._
+@import config.AppConfig
+
+@this(paragraphBody: paragraphBody,
+        link: link,
+        appConfig: AppConfig)
+
+@()(implicit messages: Messages)
+
+<div>
```

Comment:
very small comment, but does this page need this div? It doesn't seem to be doing anything, especially without a class.

Reply:
Good point, I copied this from the submission page into the component and didn't change it. Removed it now.